mautic/mautic
# Refactor dashboard to use factory and new cache provider

~~⚠️ Depends on https://github.com/mautic/mautic/pull/9107 which must be merged first! ⚠️~~

| Q  | A
| --- | ---
| Bug fix? | N
| New feature? | N
| Automated tests included? | Y
| BC breaks? | N
| Deprecations? | N

[//]: # ( Note that all new features should have a related user and/or developer documentation PR in their respective repositories. )

[//]: # ( Required: )
#### Description:
This PR makes the dashboard use the CacheBundle instead of the cache storage.

#### Steps to test this PR:
1. Enable some adapter for the CacheBundle 

For redis I recommend getting some GUI like *Redis Desktop Manager*

2. Pull this PR and clear cache
3. Refresh the /dashboard and verify that data is being written to the cache you have selected
4. Verify cache is being displayed
